### DISCUSSION TITLE
英文下怎么打词

### DISCUSSION THREAD

**Discussion by biange200** [fcitx5-android/fcitx5-android#570] [Q&A | 问答] [answered]
发现在英文下输入英文是一个一个字打的，没有单词模式？

**Comment by rocka** [comment, marked as answer]
有单词提示，会自动根据输入框的属性进行开关。如果你想无视输入框的属性强制开启单词提示，可以把 设置 - 输入法 - English - 根据编辑器属性禁用单词提示 关闭。

**Reply by biange200** [reply]
无法记住自己打过的，还有邮箱账号也无法记忆

每次都要重新打

**Reply by rocka** [reply]
目前不支持
